src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int NUMBER_OF_WORKED_HOURS_INDEX = 2;
+    private static final int AMOUNT_OF_PAYMENT_PER_HOUR_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom,
+                DateTimeFormatter.ofPattern("dd.MM.yyyy"));
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/f51b4f2c-7c92-4805-851a-37e1f0f12077)

Reply:
not fixed